contracts/interfaces/IDrugItem.sol

```diff
@@ -0,0 +1,53 @@
+pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
+
+
+contract IDrugItem {
+
+    struct Handover {
+        Participant to;
+        uint when;
+    }
+
+    struct Participant {
+        address id;
+        ParticipantCategory category;
+    }
+
+    struct TransitConditions {
+        int8 temperature;
+        TransitCategory category;
+    }
+
+    enum ParticipantCategory { Vendor, Carrier, Pharmacy }
+    enum TransitCategory { NotApplicable, Airplane, Ship, Truck }
+
+    // constructor(bytes32 _drugItemId, address _vendor, address _to, ParticipantCategory _participantCategory)
```

Comment:
Why did you leave commented code here?

Reply:
You're right. I was considering basing the task on these interface files and comments in them but firstly it would be much too complex to be done during the workshops and secondly the specification and public interfaces should be more than enough. No need to repeat ourselves.